merge_sort/CPP/mergesort.cpp

```diff
@@ -0,0 +1,57 @@
+#include <bits/stdc++.h>
+using namespace std;
+#define ll long long int
+ll a[1000005];
+ll merge(ll a[],ll x[],ll y[],ll l,ll r)
+{
+	ll i,j,k;
+	i=0;
+	j=0;
+	k=0;
+	while(i<l && j<r)
+	{
+		if(x[i]<=y[j])
+			a[k++] = x[i++];
+		else
+			a[k++] = y[j++];
+	}
+	while(i<l)
+		a[k++] = x[i++];
+	while(j<r)
+		a[k++] = y[j++];
+}
+ll mergesort(ll a[],ll n)
+{
+	if(n<2)
+		return 0;
+	ll mid = n/2;
+	ll x[mid];
+	ll y[n-mid];
+	ll i,j;
+	for(i=0;i<mid;i++)
+		x[i]=a[i];
+
+	for(j=0;j<n-mid;j++)
+		y[j] = a[i++];
+	
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmptrli6ky6/merge_sort/CPP/mergesort.cpp
+++ b/tmp/tmptrli6ky6/merge_sort/CPP/mergesort.cpp
@@ -33,7 +33,7 @@
 
 	for(j=0;j<n-mid;j++)
 		y[j] = a[i++];
-	
+
 	mergesort(x,mid);
 	mergesort(y,n-mid);
 	merge(a,x,y,mid,n-mid);
```